bigtable-client-core-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/wrappers/BigtableApi.java

```diff
@@ -27,19 +27,19 @@
  * <p>For internal use only - public for technical reasons.
  */
 @InternalApi("For internal usage only")
-public abstract class BigtableWrapper implements AutoCloseable {
+public abstract class BigtableApi implements AutoCloseable {
```

Comment:
@igorbernstein2 
Please let me know if this name sounds ok to you. Somehow I felt this sounds better. If not then I can revert back.

Also `DataClientWrapper` & `AdminClientWrapper` has getters with `BigtableApi#getDataClient` & `BigtableApi#getAdminClient`.

Reply:
Sounds good either way. I don't have strong feelings